.gitignore

```diff
@@ -142,6 +142,3 @@ toolchest_client/demo.py
 
 # Default temporary directory for splitting input files
 temp_toolchest*
-
-# Default directories for local runs of integration tests
-test_*
```

Comment:
@bcai2 this also ignored test files :P

Reply:
That doesn't seem to be the case for me -- all `.py` test files are in `tests/test_*.py` form, which doesn't match. @lebovic

Reply:
Per our convo, let's add this back and modify this to `/test_*`